framework/base/src/types/managed/wrapped/token_identifier.rs

```diff
@@ -64,6 +64,12 @@ impl<M: ManagedTypeApi> TokenIdentifier<M> {
     pub fn is_valid_esdt_identifier(&self) -> bool {
         M::managed_type_impl().validate_token_identifier(self.buffer.handle.clone())
     }
+
+    pub fn ticker(&self) -> ManagedBuffer<M> {
```

Comment:
Please also add some unit tests in `framework/scenario/tests/token_identifier_test.rs`